### PR TITLE
AMP-68986 Update other event streaming docs

### PR DESCRIPTION
**Pull request by daniel-beer**
# Amplitude Developer Docs PR

## Description

Describe your changes. 

## Deadline

When do these changes need to be live on the site?

## Change type

- [ ] Doc bug fix. Fixes #[insert issue number]. Amplitude contributors include Jira issue number. 
- [ ] Doc update.
- [ ] New documentation.
- [ ] Non-documentation related fix or update.

# PR checklist:

- [ ] My documentation follows the style guidelines of this project.
- [ ] I previewed my documentation on a local server using `mkdocs serve`.
- [ ] Running `mkdocs serve` didn't generate any failures.
- [ ] I have performed a self-review of my own documentation.

@amplitude-dev-docs

### CODE REVIEW

File: docs/data/destinations/appsflyer.md [hunk 1]
```diff
@@ -1,94 +1,91 @@
...
+
+3. (optional) In **Select additional properties**, select any more event and user properties you want to send to AppsFlyer. If you don't select any properties here, Amplitude doesn't send any. These properties are sent to AppsFlyer as [AppsFlyer event values](https://dev.appsflyer.com/hc/reference/post_s2s_inappevent). _Transformed event properties and transformed user properties aren't supported._
+
+### Enable sync
+
+When satisfied with your configuration, at the top of the page toggle the **Status** to "Enabled" and click **Save**.
+
+### Supported AppsFlyer properties
+
+_You can map Amplitude user properties directly to all the following AppsFlyer properties._
```
Comment (sina-jamshidi) [inline docs/data/destinations/appsflyer.md:73]: "directly" seems unnecessary

File: docs/data/destinations/customerio.md [hunk 2]
```diff
@@ -1,95 +1,70 @@
...
-5. Configure **Send Events** to send events ingested by Amplitude to Customer.io.
-      1. To send an event, toggle **Send Events** to **Enabled**.
-      2. Expand the **Select and filter events** panel, and select which events to send.
-      3. To select event properties to send, expand the **Specify event properties to send** panel, and select properties you want to include. If you don't select any properties here, Amplitude doesn't send any.
-6. Save when finished.
+### Configure event forwarding
 
-See the [full list of available mappings](#list-of-available-mappings).
+Under **Send Events**, make sure the toggle is enabled ("Events are sent to Customer.io") if you want to stream events to Customer.io. When enabled, events are automatically forwarded to Customer.io when they're ingested in Amplitude. Events aren't sent on a schedule or on-demand using this integration. Events are sent to Customer.io as [Customer.io events](https://www.customer.io/docs/api/track/#tag/Track-Events), including web page views and mobile screen views. Customer.io automatically creates a new user in Customer.io if the provided **Customer.io User Identifier** doesn't exist in Customer.io.
```
Comment (sina-jamshidi) [inline docs/data/destinations/customerio.md:50]: These were fine when they were shorter for other destinations but here they read more like bullet points and might be hard to follow.
Maybe bring the "considerations" heading back?
```markdown
Under **Send Events**, make sure the toggle is enabled ("Events are sent to Customer.io") if you want to stream events to Customer.io. When enabled, events are automatically forwarded to Customer.io when they're ingested in Amplitude.

#### Considerations:
- Events aren't sent on a schedule or on-demand using this integration. 
- Events are sent to Customer.io as [Customer.io events](https://www.customer.io/docs/api/track/#tag/Track-Events), including web page views and mobile screen views. 
- Customer.io automatically creates a new user in Customer.io if the provided **Customer.io User Identifier** doesn't exist in Customer.io.
```

**Review by sina-jamshidi — APPROVED** [review]

File: docs/data/destinations/appsflyer.md [hunk 3]
```diff
@@ -1,94 +1,91 @@
...
+
+| Parameter Name        | Required              | Recommended         |
+|-----------------------|:---------------------:|:-------------------:|
+| **AppsFlyer ID**      | :octicons-check-16:   |                     |
```
[the remark this answers is not in the document]
Comment (daniel-beer) [reply docs/data/destinations/appsflyer.md:77]: Yes
